pyproject.toml

```diff
@@ -15,7 +15,7 @@ packages = [{ include = "yival", from = "src" }]
 yival = 'yival.__main__:main'
 
 [tool.poetry.dependencies]
-python = ">=3.10,<3.13"
+python = ">=3.10,<3.12"
```

Comment:
We're getting low on supported versions of python, is there a way to set both langchain and llama-index to extra?

Reply:
Yes, I remeber that someone, maybe langchain llama-index or ragas, needs this change.